server/tga/author_profiles.py

```diff
@@ -116,3 +123,29 @@ def _add_cv_item_on_update(updates: Dict[str, Any], original: Dict[str, Any], cu
                 }
             )
             cv_service.patch(cv_id, cv_updates)
+
+
+def get_author_profiles_by_user_id(user_ids: List[ObjectId]) -> Dict[ObjectId, Dict[str, Any]]:
+    try:
+        service = get_resource_service("author_profiles")
+    except KeyError:
+        service = get_resource_service("items")
+
+    urn_domain = app.config["URN_DOMAIN"]
+    response = service.search(
+        {
+            "query": {
+                "bool": {
+                    "must": [
+                        {"terms": {"authors.uri": [f"urn:{urn_domain}:user:{user_id}" for user_id in user_ids]}},
+                        {"term": {"authors.role": AUTHOR_PROFILE_ROLE}},
```

Comment:
without nested this might match some extra authors, not sure if we would need to check them somehow later

Reply:
`AUTHOR_PROFILE_ROLE` (which is `"author_profile"`) is only assigned from the server and never exposed in the front end (see https://github.com/superdesk/superdesk-tga/blob/develop/server/tga/author_profiles.py#L55 for when it is applied). So we should be good here